maker/src/routes.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::let_unit_value)]
```

Comment:
🔧 Could you leave a short comment next to these statements explaining why we need them and reference the GitHub issue in the commit message?

Reply:
I'll add that, good idea!